src/NexusMods.App.UI/Services.cs

```diff
@@ -175,7 +176,13 @@ public static IServiceCollection AddUI(this IServiceCollection c, ILauncherSetti
             .AddView<FileTreeNodeView, IFileTreeNodeViewModel>()
             .AddView<LoadingView, ILoadingViewModel>()
             .AddView<ErrorView, IErrorViewModel>()
+
             .AddView<DiagnosticEntryView, IDiagnosticEntryViewModel>()
+            .AddViewModel<DiagnosticEntryViewModel, IDiagnosticEntryViewModel>()
```

Comment:
Do we need to register `DiagnosticEntryViewModel`? It needs to be constructed from code as it takes a `Diagnostic` in the constructor, is there an other reason to register it?

Reply:
This is just there to keep the services consistent. It'd look weird if we only register the View and not the View Model.